services/web/server/src/simcore_service_webserver/rest_handlers.py

```diff
@@ -82,13 +83,18 @@ async def get_config(request: web.Request):
 
 
 @routes.get(f"/{API_VTAG}/scheduled_maintenance", name="get_scheduled_maintenance")
+@login_required
 async def get_scheduled_maintenance(request: web.Request):
     """Check scheduled_maintenance table in redis"""
 
     redis_client = get_redis_scheduled_maintenance_client(request.app)
     hash_key = "maintenance"
-    # {"start": "2023-01-17T14:45:00.000Z", "end": "2023-01-17T23:00:00.000Z", "reason": "Release 1.0.4"}
+    # Examples.
+    #  {"start": "2023-01-17T14:45:00.000Z", "end": "2023-01-17T23:00:00.000Z", "reason": "Release 1.0.4"}
+    #  {"start": "2023-01-20T09:00:00.000Z", "end": "2023-01-20T10:30:00.000Z", "reason": "Release ResistanceIsFutile2"}
+    # NOTE: datetime is UTC (Canary islands / UK)
     if maintenance_str := await redis_client.get(hash_key):
         return web.json_response(data={"data": maintenance_str})
 
-    return web.json_response(status=204)
+    assert web.HTTPNoContent.status_code == 204  # nosec
```

Comment:
this is a bit weird...

Reply:
haha, well the story goes as follows. I see
```python
return web.json_response(status=204)
```
but, what was 204 ? aha, no content
```python
return web.json_response(status=web.HTTPNoContent.status_code)
```
wait, what was the status code again?
```python
return web.json_response(status=web.HTTPNoContent) # 204
```
mmhh there is a risk the comment does not get change when the status code changes
```python
assert web.HTTPNoContent.status_code == 204  # nosec
return web.json_response(status=web.HTTPNoContent)
```
well, let's connect both
```python
    response = web.json_response(status=web.HTTPNoContent.status_code)
    assert response.status == 204  # nosec
    return response
```

Reply:
starlette.status solves this dilemma nicely by including both the status name and code in enums names!